satellite/satellitedb/dbx/satellitedb.dbx

```diff
@@ -435,18 +438,33 @@ model bucket_bandwidth_rollup (
 	field settled   uint64
 )
 
+read all (
+	select bucket_bandwidth_rollup
+	where  bucket_bandwidth_rollup.project_id = ?
+	where  bucket_bandwidth_rollup.interval_start >= ?
+)
+
 model bucket_storage_rollup (
 	key    bucket_id
 	unique bucket_id interval_start interval_seconds
 
-	field bucket_id        blob
+	field bucket_name       blob
```

Comment:
Is there a way we can avoid having `bucket_name` and `project_id` here, they will end up using a lot of data.

Reply:
I agree we shouldn't store all 3 fields, since `bucket_id` duplicates the other two fields. However it seems simpler to store `bucket_name` and `project_id` in this table and join those when we need `bucketID`, versus only storing the `bucket_id` and needing to parse out the projectID when we need it.  Thoughts?

Reply:
i agree with jessica - as long as we need to keep track of a project id and a bucket name, using two separate fields won't be a ton of extra overhead. as soon as we have real honest-to-goodness foreign keys for buckets then we can do a migration, just like we would have needed to do anwyays